source/sync/configure/sync-settings.txt

```diff
@@ -173,11 +201,64 @@ Performance and Storage
 Each queryable field adds additional metadata storage to your Atlas cluster and
 may lead to degraded write performance. You should have as few queryable fields
 as needed by your application. A good rule of thumb is to have at most 10
-queryable fields.
+queryable fields. If you need to reduce storage usage or improve performance, 
+you can remove unneeded queryable fields from your App. However, be aware
+of the consequences of adding or removing queryable fields detailed below.
```

Comment:
```suggestion
of the consequences of adding or removing queryable fields detailed below in the section, "Consequences of Adding or Removing Queryable Fields".
```
So reader knows where to expect the information. Think we mostly said no to linking to sections within pages, so including the title name will at least allow reader to use TOC.